diff --git a/techpack/display/msm/sde/sde_color_processing.c b/techpack/display/msm/sde/sde_color_processing.c
index 66204c42b..e553119b4 100644
--- a/techpack/display/msm/sde/sde_color_processing.c
+++ b/techpack/display/msm/sde/sde_color_processing.c
@@ -1560,9 +1560,9 @@ static void sde_cp_crtc_setfeature(struct sde_cp_node *prop_node,
 	bool feature_enabled = false;
 	struct sde_mdss_cfg *catalog = NULL;
 	struct drm_crtc *drm_crtc = &sde_crtc->base;
-	struct sde_crtc_state *cstate = to_sde_crtc_state(drm_crtc->state);
-	struct drm_property_blob *blob;
-	struct drm_msm_pcc *pcc_cfg;
+	__maybe_unused struct sde_crtc_state *cstate = to_sde_crtc_state(drm_crtc->state);
+	__maybe_unused struct drm_property_blob *blob;
+	__maybe_unused struct drm_msm_pcc *pcc_cfg;
 
 	memset(&hw_cfg, 0, sizeof(hw_cfg));
 	sde_cp_get_hw_payload(prop_node, &hw_cfg, &feature_enabled);
